### ISSUE TITLE
Error when trying to build project.

### ISSUE THREAD

**Issue by TheAndrew2115** [Autodesk/sydevs#7]
Hello,

I am trying to create my own SyDEVS project, but I seem to be running into an issue with building my project. To get started, I am following the tutorial's instructions on how to set up my project (Part 1), but when I type the command:

_cmake -G "Xcode" ..               or
cmake  .._

I get an error:

_CMake Error at CMakeLists.txt:57 (add_library):
  No SOURCES given to target: SyDEVS-static

CMake Generate step failed.  Build files cannot be regenerated correctly._

I'm sure I built the directory structure and the CMakeLists.txt file the exact same way the tutorial did (aside from the overall project folder's name) so I'm pretty confused as to why I'm getting this error.

For reference, this is line 57 of my CMakeLists.txt file:

_add_library(SyDEVS-static STATIC ${CORE_SRCS} ${TIME_SRCS} ${SYSTEMS_SRCS} ${CORE_HDRS} ${TIME_HDRS} ${SYSTEMS_HDRS})_

**Comment by rhysgoldstein** [comment]
Hi Andrew, Can you insert the following line into CMakeLists.txt, right before the offending line?

`message(${CORE_SRCS})`

Then when you run `cmake ..` again from the `build` directory, does it print the following?

`external/sydevs-0.6.6/src/sydevs/core/scale.cpp`

**Comment by TheAndrew2115** [comment]
Hey Rhys, thanks for the response! I managed to fix the silly issue.

It didn't print external/sydevs-0.6.6/src/sydevs/core/scale.cpp, instead I got 
`message called with incorrect number of arguments`

When I downloaded the sydevs-0.6.6 folder for the second time it added a '2' to the end of its name. I simply had to rename it sydevs-0.6.6 and the issue was fixed.

Thanks again for the response!
